packages/data/addon/services/keg.ts

```diff
@@ -85,6 +91,51 @@ export default class KegService extends Service {
     return this.pushMany(type, [rawRecord], options).firstObject;
   }
 
+  /**
+   * Inserts a single record into the store
+   *
+   * @param type - name of the model type
+   * @param record - record to be inserted into the keg
+   * @param options - config object
+   */
+  insert(type: string, record: KegRecord, options: Partial<InsertOptions> = {}): KegRecord {
+    return this.insertMany(type, [record], options).firstObject as KegRecord;
+  }
+
+  /**
+   * Inserts an array of records into the store
+   *
+   * @param type - type name of the model type
+   * @param record - record to be inserted into the keg
+   * @param options - config object
+   */
+  insertMany(type: string, records: Array<KegRecord>, options: Partial<InsertOptions> = {}): EmberArray<KegRecord> {
```

Comment:
Is pushMany being deprecated or is there a significant difference between this and that?

Reply:
`insert` does not populate a model (`model.create(payload)`, that is the main diff. But we will deprecated push methods at some point. Maybe when we remove keg as a service?